docs/core/tools/dotnet-store.md

```diff
@@ -0,0 +1,87 @@
+---
+title: dotnet store command | Microsoft Docs
+description: The 'dotnet store' command stores the specified assemblies in the runtime package store.
+keywords: dotnet store, dotnet store, CLI, CLI command, .NET Core
+author: bleroy
+ms.author: mairaw
+ms.date: 06/26/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: 1e8e4122-8110-4b48-afce-afffb6737776
+---
+
+# dotnet store
+
+[!INCLUDE [core-preview-warning](~/includes/core-preview-warning.md)]
+
+## Name
+
+`dotnet store` - Stores the specified assemblies in the [runtime package store](../deploying/runtime-package-store.md).
+
+## Synopsis
+
+`dotnet store -m|--manifest -f|--framework -r|--runtime  [--framework-version] [-h|--help] [--output] [--skip-optimization] [--skip-symbols] [-v|--verbosity] [--working-dir]`
+
+## Description
+
+`dotnet store` stores the specified assemblies in the [runtime package store](../deploying/runtime-package-store.md). By default, assemblies are optimized for the target runtime and framework. For more information, see the [runtime package store](../deploying/runtime-package-store.md) topic.
+
+## Required options
+
+`-f|--framework <FRAMEWORK>`
```

Comment:
@mairaw Sounds good to me, but isn't that a separate issue for discussion with sample renderings? Would u like me to open an issue for it?